suite-native/module-settings/src/screens/TurnOffFirmwareAuthencityCheckModalScreen.tsx

```diff
@@ -0,0 +1,99 @@
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { TouchableOpacity } from 'react-native';
+
+import { useNavigation } from '@react-navigation/native';
+
+import {
+    SettingsStackParamList,
+    SettingsStackRoutes,
+    Screen,
+    ScreenHeader,
+    StackNavigationProps,
+} from '@suite-native/navigation';
+import {
+    Box,
+    Button,
+    CheckBox,
+    IconListItem,
+    Text,
+    HStack,
+    VStack,
+    Card,
+} from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import { setCheckFirmwareAuthenticityDisabled } from '@suite-native/settings';
+
+const InformativeList = () => (
+    <VStack spacing="sp24">
+        <IconListItem icon="warning" variant="yellow" iconSize="large" verticalAlign="flex-start">
+            <Text variant="highlight">
+                <Translation id="moduleSettings.deviceChecks.firmwareAuthenticityCheck.turnOffModal.item1" />
+            </Text>
+            <Text>
+                <Translation id="moduleSettings.deviceChecks.firmwareAuthenticityCheck.turnOffModal.item1Explanation" />
+            </Text>
+        </IconListItem>
+        <IconListItem icon="code" variant="yellow" iconSize="large" verticalAlign="flex-start">
+            <Text variant="highlight">
+                <Translation id="moduleSettings.deviceChecks.firmwareAuthenticityCheck.turnOffModal.item2" />
+            </Text>
+            <Text>
+                <Translation id="moduleSettings.deviceChecks.firmwareAuthenticityCheck.turnOffModal.item2Explanation" />
+            </Text>
+        </IconListItem>
+    </VStack>
+);
+
+type NavigationProp = StackNavigationProps<
+    SettingsStackParamList,
+    SettingsStackRoutes.SettingsDeviceChecks
+>;
+
+export const TurnOffFirmwareAuthencityCheckModalScreen = () => {
```

Comment:
```suggestion
export const TurnOffFirmwareAuthenticityCheckModalScreen = () => {
```

fix the file name as well.

Reply:
952bf18977652b5ca5fe79f3e4ae4fac20ae413c